azure-pipelines/official.yml

```diff
@@ -4,6 +4,18 @@ trigger:
   paths:
     exclude: ["*.md"]
 
+schedules:
+  - cron: "0 8 23-29 * 0"
```

Comment:
nit: add a comment summarizing the cron syntax for readability so it doesn't require a trip to the docs each time 😉

Reply:
Added, and tweaked the dates so it is "4th Sunday" consistently